client/app/app.vue

```diff
@@ -18,14 +24,100 @@
                         svg: use(:xlink:href="require('./assets/sprites/icon-connected.svg')")
                         | {{ 'my_accounts connected title' | t }}
 
-        router-view
+        router-view(v-on:open-dialog='onOpenDialog')
+
 </template>
 
 
+<script>
+    const dialogs = [{
+        id: 'dialog-1',
+        hidden: false,
+        headerImage: 'test0.png',
+        success: {
+            route: { name: 'create-account-success' }
+        }
+    }]
+
+
+    export default {
+      data() {
+          return {
+              'dialogs': []
+          }
+      },
+
+      created () {
+          const query = this.$router.currentRoute.query
+
+          // Show Dialogs
+          if (query.dialog) {
+              const values = query.dialog.split(',')
+              values.forEach((value) => {
+                  this.onOpenDialog(value)
+              })
+          }
+      },
+
+      methods: {
+          onError (err) {
+
```

Comment:
@m4dz : Should we use `Vue.config.errorHandler` here or is it a little "too much" ? https://vuejs.org/v2/api/#errorHandler